.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,23 @@
+name: Lint Python
```

Comment:
Rename to `lint-python.yml` to be consistent with the other workflows.

Reply:
Done.